pkg/tinydtls/patches/0002-fix-printf-warning-for-64-bit-systems.patch

```diff
@@ -0,0 +1,27 @@
+From df0fd3c00918f1698a87c2786590dcd097ee0625 Mon Sep 17 00:00:00 2001
+From: Frederik Haxel <haxel@fzi.de>
+Date: Mon, 14 Aug 2023 09:54:35 +0200
+Subject: [PATCH] Fix printf warning for 64 bit systems
+
+---
+ dtls_debug.h | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/dtls_debug.h b/dtls_debug.h
+index 5b58acb..3bf04f0 100644
+--- a/dtls_debug.h
++++ b/dtls_debug.h
+@@ -139,8 +139,8 @@ void dtls_dsrv_log_addr(log_t level, const char *name, const session_t *addr);
+ #define dtls_notice(...) LOG_INFO(__VA_ARGS__)
+ #define dtls_info(...) LOG_INFO(__VA_ARGS__)
+ #define dtls_debug(...) LOG_DEBUG(__VA_ARGS__)
+-#define dtls_debug_hexdump(name, buf, length) { if (LOG_DEBUG <= LOG_LEVEL) { LOG_DEBUG("-= %s (%zu bytes) =-\n", name, length); od_hex_dump(buf, length, 0); }}
+-#define dtls_debug_dump(name, buf, length) { if (LOG_DEBUG <= LOG_LEVEL) { LOG_DEBUG("%s (%zu bytes):", name, length); od_hex_dump(buf, length, 0); }}
++#define dtls_debug_hexdump(name, buf, length) { if (LOG_DEBUG <= LOG_LEVEL) { LOG_DEBUG("-= %s (%zu bytes) =-\n", name, (size_t) length); od_hex_dump(buf, length, 0); }}
++#define dtls_debug_dump(name, buf, length) { if (LOG_DEBUG <= LOG_LEVEL) { LOG_DEBUG("%s (%zu bytes):", name, (size_t) length); od_hex_dump(buf, length, 0); }}
```

Comment:
We usually get our patches merged upstream for this quite well, so please try to open a PR there. (Reminds me that I can delete `0001-dtls_debug-mock-dsrv_log-when-NDEBUG.patch` now since that one [got merged](https://github.com/eclipse/tinydtls/pull/189))

Reply:
PR created.